src/dataset.py

```diff
@@ -44,15 +53,21 @@ def __getitem__(self, i):
             cropped_label_path = Path(self.base_dir, cropped_label_path)
 
             cropped_img = np.load(cropped_img_path)
-            mask_path = cropped_label_path if (
-                self.mode == 'train') else mask_path
+            mask_path = cropped_label_path
             image_path = cropped_img_path
 
         img = np.load(image_path)
         img = T.ToTensor()(img)
 
-        if self.channels_to_inc is not None:
-            img = img[self.channels_to_inc]
+        # get snow index before filtering the data
+        snow_i = utils.get_snow_index(img)
```

Comment:
why not just call it `snow_index` from here, so that you don't need to redefine it later?

Reply:
Just didn't notice. Thanks